### ISSUE TITLE
Illegal code in Utilities

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#8]
**Reported by HansOlsson on 14 Nov 2007 10:51 UTC**
The code in http://trac.modelica.org/MSL/browser/trunk/Modelica/Utilities/Examples.mo

has

```
function expression
**protected**  
**encapsulated** function term "Evaluate term of an expression" 
import Modelica.Utilities.Examples.expression.primary;
... 
end term;
**encapsulated** function primary ...
end primary;
...
end expression;
```

This uses import of item that is not in a package, and furthermore the item is protected. Remove 'encapsulated', and 'import' and it will automatically work.

Issue is also present in 2.2.2

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/8

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 16 Nov 2007 08:36 UTC**

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 16 Nov 2007 08:37 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 16 Nov 2007 11:38 UTC**
fixed in fb3f8466e523d181536965522c576ec09aa88e18, 4d9cbefa36748b5fa7397cb23fa2018a3236e00c
